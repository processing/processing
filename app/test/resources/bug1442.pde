float a() {
	return 1.0;
}