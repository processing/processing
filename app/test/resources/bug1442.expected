import processing.core.*; 
import processing.data.*; 
import processing.event.*; 
import processing.opengl.*; 

import java.util.HashMap; 
import java.util.ArrayList; 
import java.io.File; 
import java.io.BufferedReader; 
import java.io.PrintWriter; 
import java.io.InputStream; 
import java.io.OutputStream; 
import java.io.IOException; 

public class bug1442 extends PApplet {

public float a() {
	return 1.0f;
}
    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "bug1442" };
        if (passedArgs != null) {
          PApplet.main(concat(appletArgs, passedArgs));
        } else {
          PApplet.main(appletArgs);
        }
    }
}
